package.json

```diff
@@ -112,7 +113,9 @@
     "uuid": "9.0.1",
     "webpack": "^4.35.3",
     "webpack-cli": "^3.3.0",
-    "webpack-dev-server": "^3.2.1",
-    "lint-staged": "^14.0.1"
+    "webpack-dev-server": "^3.2.1"
+  },
+  "dependencies": {
+    "@types/jsonwebtoken": "^9.0.6"
```

Comment:
Why are the types added as dependencies here and not in the package.json of the new MS?